tower/src/balance/p2c/service.rs

```diff
@@ -94,10 +85,10 @@ where
     D::Service: Service<Req>,
     <D::Service as Service<Req>>::Error: Into<crate::BoxError>,
 {
-    /// Initializes a P2C load balancer from the provided randomization source.
-    pub fn new(discover: D, rng: SmallRng) -> Self {
+    /// Construct a load balancer that uses operating system entropy.
```

Comment:
Nit/TIOLI: I think these are usually written in the form
```suggestion
    /// Constructs a load balancer that uses operating system entropy.
```